src/data/markdown/docs/03 cloud/09 Optional Add-ons/01 Private load zones.md

```diff
@@ -0,0 +1,72 @@
+---
+title: 'Private load zones'
+excerpt: 'How to set up private load zones in k6 Cloud'
+---
+
+A private load zone is a special location that we set up for you on k6 Cloud. This location corresponds to availability zones you've selected on your own AWS account. With private load zones, you can start tests on k6 Cloud but have them execute on _your own cloud infrastructure_, provisioned within AWS.
+
+## Benefits of using private load zones
+
+Here are some reasons why you might want to use a private load zone on k6 Cloud:
+- Improved privacy or application security while doing load testing
+- Ability to leverage k6 Cloud's unified dashboards and visualization features
+- Reuse of existing cloud provider account to provision the infrastructure
+- Improved visibility into instance usage and provisioning activity on your team
+
+## How do you use a private load zone?
+
+First, [contact us](mailto:support@k6.io) about adding private load zones to your subscription. We will give you information about how to proceed based on your requirements, including the setup that is required on your end to allow k6 Cloud to start instances on your behalf.
+
+Second, once we've confirmed that your private load zone is ready to go, set up your test to select the private load zone. You can do this one of two ways:
+- via the [graphical Test Builder](https://k6.io/docs/test-authoring/test-builder) on k6 Cloud
+- via the k6 script, within the [test options](https://k6.io/docs/using-k6/options/)
+
+Here's how to define a private load zone in the test options of your k6 script:
+
+```javascript
+export const options = {
+  ext: {
+    loadimpact: {
+      distribution: {
+        private: {
+          loadZone: 'amazon:eu:private load zone',
+          percent: 100,
+        },
+      },
+    },
+  },
+};
+```
+
+
+Third, [start the test](https://k6.io/docs/cloud/creating-and-running-a-test/test-authoring/#running-a-cloud-test) by:
```

Comment:
I would not specify how to run a cloud test.  

Mention only that is triggered as any other cloud test (including the link) to reinforce the simplicity of this aspect.

Reply:
Good point. Addressed in the latest commit!